api/app.go

```diff
@@ -123,15 +125,30 @@ func (app *App) configureNewRelic() error {
 	return nil
 }
 
+func (app *App) configureJaeger() {
+	l := app.Logger.With(
+		zap.String("source", "app"),
+		zap.String("operation", "configureJaeger"),
+	)
+
+	opts := jaeger.Options{
+		Disabled:    false,
+		Probability: 1.0,
+		ServiceName: "podium",
+	}
+
+	_, err := jaeger.Configure(opts)
+	if err != nil {
+		l.Error("Failed to initialize Jaeger.")
+	}
+}
+
 func (app *App) setConfigurationDefaults() {
 	app.Config.SetDefault("healthcheck.workingText", "WORKING")
 	app.Config.SetDefault("api.maxReturnedMembers", 2000)
 	app.Config.SetDefault("api.maxReadBufferSize", 32000)
-	app.Config.SetDefault("redis.host", "localhost")
-	app.Config.SetDefault("redis.port", 1212)
-	app.Config.SetDefault("redis.password", "")
-	app.Config.SetDefault("redis.db", 0)
-	app.Config.SetDefault("redis.maxPoolSize", 20)
+	app.Config.SetDefault("redis.url", "redis://localhost:1212/0")
```

Comment:
for backwards compatibility you should build the url from the available variables (host, port, etc)

Reply:
😵